go.mod

```diff
@@ -18,8 +18,7 @@ require (
 	github.com/docker/go-units v0.4.0
 	github.com/go-sql-driver/mysql v1.6.0
 	github.com/gofrs/uuid v4.2.0+incompatible
-	github.com/golang-collections/collections v0.0.0-20130729185459-604e922904d3
-	github.com/golang/glog v1.1.0
+	github.com/golang/glog v1.2.4
```

Comment:
Can you merge the origin into this? I believe this change was already done in previous commit #4182

Reply:
Yes, it was already done in previous commit

Reply:
go.mod updated and  in right state now

Reply:
![image](https://github.com/user-attachments/assets/07768c32-9bed-4f43-acef-ee297767d519)

Why does this show a change in YOUR PR?